superset-frontend/packages/superset-ui-core/src/chart/components/SuperChartCore.tsx

```diff
@@ -89,27 +89,76 @@ export default class SuperChartCore extends PureComponent<Props, {}> {
    * and return previous value
    * unless one of
    * - preTransformProps
-   * - transformProps
-   * - postTransformProps
    * - chartProps
    * is changed.
    */
-  processChartProps = createSelector(
+  preSelector = createSelector(
     [
       (input: {
         chartProps: ChartProps;
         preTransformProps?: PreTransformProps;
-        transformProps?: TransformProps;
-        postTransformProps?: PostTransformProps;
       }) => input.chartProps,
       input => input.preTransformProps,
+    ],
+    (chartProps, pre = IDENTITY) => pre(chartProps),
+  );
+
+  /**
```

Comment:
Could you adjust the comments for `transformSelector` and `postSelector` to:

> Memoized the function so it will not recompute and return the previous value unless one of the input arguments have changed.